src/ErrorHandler.php

```diff
@@ -357,18 +357,19 @@ public function addExceptionHandlerListener(callable $listener): void
      * Handles errors by capturing them through the Raven client according to
      * the configured bit field.
      *
-     * @param int    $level   The level of the error raised, represented by one
-     *                        of the E_* constants
-     * @param string $message The error message
-     * @param string $file    The filename the error was raised in
-     * @param int    $line    The line number the error was raised at
+     * @param int    $level      The level of the error raised, represented by
+     *                           one of the E_* constants
+     * @param string $message    The error message
+     * @param string $file       The filename the error was raised in
+     * @param int    $line       The line number the error was raised at
+     * @param array  $errcontext The error context (deprecated in PHP 7.2)
```

Comment:
I think that it's more correct to say `deprecated since ...` instead of `deprecated in`